exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/grpc/GrpcExporterUtil.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.internal.grpc;
+
+import io.grpc.ManagedChannel;
+import io.opentelemetry.exporter.otlp.internal.Marshaler;
+import java.net.URI;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+final class GrpcExporterUtil {
+
+  private static final boolean USE_OKHTTP;
+
+  static {
+    boolean useOkhttp = false;
+    // Use the OkHttp exporter if ManagedChannel is not found and OkHttp is.
+    try {
+      Class.forName("io.grpc.ManagedChannel");
+    } catch (ClassNotFoundException e) {
+      try {
+        Class.forName("okhttp3.OkHttpClient");
+        useOkhttp = true;
+      } catch (ClassNotFoundException classNotFoundException) {
+        // Fall through
+      }
+    }
+    USE_OKHTTP = useOkhttp;
+  }
+
+  static <T extends Marshaler> GrpcExporterBuilder<T> exporterBuilder(
+      String type,
+      long defaultTimeoutSecs,
```

Comment:
#nit: Default timeout is always the same, no? Could omit this argument.

Reply:
Yeah I don't think we can lose the reference in the javadoc though for users in the public API like 

https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/otlp/trace/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterBuilder.java#L52

Could move the constant to a shared place and reference it, but this seemed more straightforward.